node/src/main/scala/org/bitcoins/node/NeutrinoNode.scala

```diff
@@ -41,13 +42,24 @@ case class NeutrinoNode(
     copy(dataMessageHandler = dataMessageHandler)
   }
 
+  lazy val randomPeerMsgSenderWithCompactFilters: PeerMessageSender = {
```

Comment:
this should be a `def` so we get a different peer every time

Reply:
Ok, changing that. I was under the impression that we we would need to have same peer for getcfheaders and for syncing. Also, looking into data message handler, missed that one.